runtime/lua/vim/_watch.lua

```diff
@@ -227,11 +227,12 @@ end
 --- @param data string
 --- @param opts vim._watch.Opts?
 --- @param callback vim._watch.Callback
-local function fswatch_output_handler(data, opts, callback)
+local function inotifywait_output_handler(data, opts, callback)
```

Comment:
```suggestion
local function on_inotifywait_output(data, opts, callback)
```

Reply:
Thanks. I changed it and pushed a new version.